static/css/section/_management.scss

```diff
@@ -1,428 +1,274 @@
-body.management-working-with-landscape {
-/*	.row {
-		padding-bottom: 60px;
-	}
-	.row-hero {
-		padding-bottom: 40px;
-	}*/
-	.row-hero .eight-col {
-		margin-bottom: 0;
-		padding-bottom: 1.25em;
-	}
-	//.row .six-col { margin-bottom: 0; }
-
-	iframe { margin-bottom: 1.25em; }
-
-	.row-landscape-trial {
-		padding-bottom: 0;
-		border-bottom: 0;
-	}
-}
+  .screenshot {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+  }
 
-body.management-ema-whitepaper {
-	.row iframe {
-		width: 100%;
-	}
+  .inventory {
+    margin-bottom: 30px;
+  }
 }
 
-/* end @section desktop management */
+// @subsection /management/ubuntu-advantage
+.management-ubuntu-advantage {
 
-/* Medium / Tablet viewport
--------------------------------------------------------------- */
-@media only screen and (max-width : 768px) {
-	body.management-ubuntu-advantage {
-		.equal-height > li {
-			height: auto !important;
-		}
-	}
-}
+  .ua-price {
+    font-size: 2em;
+  }
+
+  .row-cloud-management {
+    border-bottom: 0;
+  }
+
+  .row-pricing {
 
-/* Medium / Tablet viewport
--------------------------------------------------------------- */
-@media only screen and (min-width : 768px) {
-	body.management {
-		.stat dl {
-      margin: 0 0 0 40px;
-      float: left;
+    @media only screen and (max-width: $breakpoint-medium) {
+      &__image {
+        float: left;
+        width: 50px;
+      }
+
+      &__image-container {
+        float: left;
+        margin-bottom: 0;
+        margin-right: 1em;
+        width: 50px;
+      }
+
+      &__title {
+        margin: .3em 1em 1.8em;
+      }
     }
-		.stats-panel dd, .stat dd {
-      margin-left: 0;
-		}
-		.stats-panel p a {
-      font-size: .8125em;
+  }
+
+  .equal-height > li {
+    @media only screen and (max-width : $breakpoint-medium) {
+      height: auto;
     }
-		.row-return-investment {
-			background-image: url("#{$asset-server}d600dad9-image-cutyouritcosts.svg");
-			background-position: 96px bottom;
-			background-repeat: no-repeat;
-		}
-		img.flush-right {
-			float: right;
-			margin-right: -40px;
-			margin-left: 20px;
-		}
-		img.flush-left {
-			float: left;
-			margin-left: -40px;
-		}
-	}
-
-	html.no-svg,
-	html.opera-mini {
-		.row-return-investment {
-			background-image: url("#{$asset-server}d1dbbf8e-image-cutyouritcosts.png");
-		}
-	}
-
-	body.management-home {
-		.margin-top { margin-top: 20px; }
-	}
-
-	.three-divided li {
-		width: 30%;
-		padding-bottom: 38px;
-	}
-
-	.row-cloud-management .three-divided li {
-    padding-bottom: 0;
-	}
-
-	body.management-ubuntu-advantage {
-		.price {
-			margin-bottom: 0;
-		}
-	}
-
-	html.no-svg,
-	html.opera-mini {
-		body.management .row-return-investment {
-			background-image: url("#{$asset-server}d1dbbf8e-image-cutyouritcosts.png");
-		}
-	}
-	body.management-ubuntu-advantage,
-	body.management-working-with-landscape {
-		.row-hero .link-cta-canonical {
-			margin-bottom: 0;
-		}
-	}
+  }
+
+  .ubuntu-advantage {
```

Comment:
This cant be seelected. As it is on the body class which is selected above. This is not effecting anything. Can you remove please.